kebab.json

```diff
@@ -0,0 +1,28 @@
+{
+    "response": [
+        {
+            "maintainer": "Bojan Tanasic (bojantanasic)",
+            "oem": "OnePlus",
+            "device": "OnePlus 8T",
+            "filename": "crDroidAndroid-16.0-20250905-kebab-v12.0.zip",
+            "download": "https://sourceforge.net/projects/crdroid/files/kebab/12.x/crDroidAndroid-16.0-20250905-kebab-v12.0.zip/download",
+            "timestamp": 1757045500,
+            "md5": "d67156d7aff71db74d6170ff1c58b297",
+            "sha256": "7e82da72a3b1e9a897c95a3e84c635b42555460b278a7dc9f7f1b9585bf58d61",
+            "size": 1598810316,
+            "version": "12.0",
+            "buildtype": "Monthly",
+            "forum": "https://xdaforums.com/t/rom-14-official-8t-crdroid-v10-8-14-09-2024.4692700/",
```

Comment:
fix forum links:
- features: https://github.com/crdroidandroid/crdroid_features/blob/16.0/README.mkdn
- rom download link (should point to 12): https://crdroid.net/kebab/12
- changelog: https://crdroid.net/kebab/12#changelog
- recovery
- crDroid Device Group and Support Community: https://crdroid.net/kebab/12/support